numpyro/diagnostics.py

```diff
@@ -25,7 +26,7 @@
 ]
 
 
-def _compute_chain_variance_stats(x: np.ndarray) -> tuple[np.ndarray, np.ndarray]:
+def _compute_chain_variance_stats(x: npt.NDArray) -> tuple[npt.NDArray, npt.NDArray]:
```

Comment:
why NDArray is used instead of ndarray?

Reply:
MyPy started complaining while I was adding hints and decided to follow the typing guidelines from numpy https://numpy.org/doc/stable/reference/typing.html :) Then all errors were gone

Reply:
interesting. thanks! maybe import `NDArray` instead of `npt`? It makes the code easier to read.

Reply:
Sure! changed in https://github.com/pyro-ppl/numpyro/pull/1940/commits/1bd90ea119bb3fcb0faa8068fa2ee48707d0334f